src/output/grid_details.rs

```diff
@@ -90,6 +91,8 @@ pub struct Render<'a> {
     pub console_width: usize,
 
     pub git_repos: bool,
+
+    pub quotes: Quotes,
```

Comment:
I was going to say that having this be a reference
```rust
pub quotes: &'a Quotes
```
and then passing it around by value by dereferencing `*self.quotes` would be the best idea.. but I think it might be more efficient to store it on the field and pass it around by value since it implements `copy`